serving/src/main/java/feast/serving/configuration/JobServiceConfig.java

```diff
@@ -16,48 +16,64 @@
  */
 package feast.serving.configuration;
 
+import feast.core.StoreProto;
 import feast.core.StoreProto.Store.StoreType;
 import feast.serving.FeastProperties;
 import feast.serving.service.JobService;
 import feast.serving.service.NoopJobService;
 import feast.serving.service.RedisBackedJobService;
 import feast.serving.specs.CachedSpecService;
-import java.util.Map;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.RedisURI;
+import io.lettuce.core.api.StatefulRedisConnection;
+import io.lettuce.core.codec.ByteArrayCodec;
+import io.lettuce.core.resource.ClientResources;
+import org.springframework.beans.factory.ObjectProvider;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import redis.clients.jedis.JedisPool;
-import redis.clients.jedis.JedisPoolConfig;
+
+import java.util.Map;
 
 @Configuration
 public class JobServiceConfig {
 
-  public static final String DEFAULT_REDIS_MAX_CONN = "8";
-  public static final String DEFAULT_REDIS_MAX_IDLE = "8";
-  public static final String DEFAULT_REDIS_MAX_WAIT_MILLIS = "50";
+  @Bean
+  Map<String, String> jobRedisConf(FeastProperties feastProperties) {
+    if (feastProperties.getJobs() == null)
```

Comment:
Is this method appropriately named if you are returning `feastProperties.getJobs().getStoreOptions()`? Couldn't that store option be a different store